osmtm/templates/task.history.mako

```diff
@@ -73,6 +73,10 @@ import bleach
 
     <p class="text-muted">
       <em title="${step.date}Z" class="timeago"></em>
+      % if isinstance(step, TaskLock) and step.duration:
+      <em>${_('during')}</em>
```

Comment:
I agree. I hope this will not lead to confusion for translators.
I'll have to figure out how "contexts" work in transifex some day.